<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcAddCardWidget</class>
 <widget class="QWidget" name="tcAddCardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QTextEdit" name="teFront">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="btnClose">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnClearBoxText">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear Both Boxes</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="btnAdd">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Front:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextEdit" name="teBack">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choose Section:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSections">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbSections</tabstop>
  <tabstop>btnClearBoxText</tabstop>
  <tabstop>teFront</tabstop>
  <tabstop>teBack</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
